usage_examples/streamlit/streamlit_app.py

```diff
@@ -0,0 +1,32 @@
+from spacy import util
+from spacy_streamlit import visualize_ner, visualize_textcat
+import streamlit as st
+
+DEFAULT_TEXT = "Matt Honnibal is the CEO of Explosion."
```

Comment:
Let's pick a different example sentence?

Reply:
Is the one I changed it to better? Suggestions are welcome

Reply:
Better! (Still slightly old-white-dude for my taste, but it's fine if it works for both the demo cases.)